r/tests/testthat/test-dplyr-funcs-datetime.R

```diff
@@ -382,6 +382,15 @@ test_that("extract epiweek from timestamp", {
   )
 })
 
+test_that("extract week from timestamp", {
```

Comment:
This looks good!  Would you mind also adding a test for "extract week from date"?